src/domain/schemas/document-schemas.js

```diff
@@ -52,3 +52,55 @@ export const hardDeleteSectionBodySchema = joi.object({
 export const hardDeleteDocumentBodySchema = joi.object({
   documentKey: idOrKeySchema.required()
 });
+
+const sectionDBSchema = joi.object({
+  revision: joi.string().required(),
```

Comment:
You can use the `idOrKeySchema` also here

Reply:
True, used!